CMakeLists.txt

```diff
@@ -13,33 +13,35 @@ include_directories(
   ${catkin_INCLUDE_DIRS}
 )
 
-execute_process(
-  WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/${PROJECT_NAME}
-  COMMAND mkdir -p www
+# Copy needed files to www build folder
+file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/www)
+configure_file(
+  ${CMAKE_CURRENT_SOURCE_DIR}/index.html
+  ${CMAKE_CURRENT_BINARY_DIR}/www/index.html
```

Comment:
> `configure_file` docs:
> If <input> is a relative path it is evaluated with respect to the current source directory.
> If <output> is a relative path it is evaluated with respect to the current binary directory.

You can remove the leading `${CMAKE_CURRENT_SOURCE_DIR}/` and `${CMAKE_CURRENT_BINARY_DIR}/`.

Same below.

Reply:
Thanks, removed in 64cf00da7be27b9dc567fa5bf8b78092e97e68e4